src/formatting/string.rs

```diff
@@ -268,6 +268,24 @@ fn break_string(max_width: usize, trim_end: bool, line_end: &str, input: &[&str]
         }
     };
 
+    // Don't allow break at `\` as it is used to concat string lines.
+    let break_not_at_backslash = |index /* grapheme at index is included */| {
+        if input[0..=index].iter().last().unwrap().ne(&"\\") {
```

Comment:
OK. Found that explicit casting should be added: `(input[index] as &str)`.  Otherwise there is an error `cannot infer type`, but I assume adding the casting is o.k.